File: WdtcCore/src/main/java/org/wdt/wdtc/core/manger/URLManger.java
package org.wdt.wdtc.core.manger;

import org.wdt.wdtc.core.download.infterface.DownloadSourceInterface;

public class URLManger {
  public static final String BMCALAPI_COM = "https://download.mcbbs.net/";
  public static final String ALIYUN_MAVEN = "https://maven.aliyun.com/repository/public/";


  public static String getPistonDataMojang() {
    return DownloadSourceInterface.PISTON_DATA_MOJANG;
  }

  public static String getPistonMetaMojang() {
    return DownloadSourceInterface.PISTON_META_MOJANG;
  }

  public static String getLittleskinUrl() {
    return "https://littleskin.cn";
  }


  public static String getMojangLibraries() {
    return DownloadSourceInterface.MOJANG_LIBRARIES;
  }

  public static String getLittleskinApi() {
    return getLittleskinUrl() + "/api/yggdrasil";
  }


}

File: WdtcCore/src/main/kotlin/org/wdt/wdtc/core/manger/URLManger.kt
@file:JvmName("URLManger")

package org.wdt.wdtc.core.manger

import org.wdt.wdtc.core.download.infterface.DownloadSourceInterface
import org.wdt.wdtc.core.utils.toURL
import java.net.URL

const val BMCALAPI_COM = "https://download.mcbbs.net/"
const val ALIYUN_MAVEN = "https://maven.aliyun.com/repository/public/"

const val pistonDataMojang: String = DownloadSourceInterface.PISTON_DATA_MOJANG

const val pistonMetaMojang: String = DownloadSourceInterface.PISTON_META_MOJANG

const val littleskinUrl: String = "https://littleskin.cn"

const val mojangLibrariesUrl: String = DownloadSourceInterface.MOJANG_LIBRARIES

val littleskinApiUrl: URL = "$littleskinUrl/api/yggdrasil".toURL()

